packages/rest/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -701,7 +702,16 @@ describe('Routing', () => {
         .expect(200, 'hello');
     });
 
-    it('sorts routes based on their specifics', async () => {
+    it('allows pluggable router', async () => {
+      const app = new RestApplication();
+      app.bind(RestBindings.ROUTER).toClass(TrieRouter);
+      const server = await app.getServer(RestServer);
+      const handler = await server.get(RestBindings.HANDLER);
+      // tslint:disable-next-line:no-any
+      expect((handler as any)._routes._router).to.be.instanceof(TrieRouter);
```

Comment:
nitpick: why do we need to cast handler as `any` here? can we do `const handler = await server.get<HttpHandler>(RestBindings.HANDLER);` above?

Reply:
It's a hacky way to test non-public properites.

Reply:
@raymondfeng Please add a comment to the code to preserve this explanation for future readers.